### PR TITLE
feat(integrations): add support for whatsapp business

### PR DESCRIPTION
**Pull request by hassan254-prog**
## Describe your changes

- Add support for Whatsapp Business

## Issue ticket number and link
[EXT-160](https://linear.app/nango/issue/EXT-160/add-support-for-whatsapp)

### CODE REVIEW

File: docs-v2/integrations/all/whatsapp-business.mdx [hunk 1]
```diff
@@ -0,0 +1,30 @@
+---
+title: Whatsapp Business
```
Comment (bodinsamuel) [inline docs-v2/integrations/all/whatsapp-business.mdx]: WhatsApp

File: packages/shared/providers.yaml [hunk 2]
```diff
@@ -5487,6 +5487,25 @@ webflow:
...
+    display_name: whatsapp
+    categories:
+        - communication
+        - marketing
+    auth_mode: API_KEY
+    proxy:
+        headers:
+            Authorization: Bearer ${apiKey}
+            Content-Type: application/json
+        base_url: https://graph.facebook.com
```
Comment (bodinsamuel) [inline packages/shared/providers.yaml:5720]: could add a test no?
e.g: this seems safe https://developers.facebook.com/docs/graph-api/reference/whats-app-business-account/product_catalogs/

Comment (hassan254-prog) [reply packages/shared/providers.yaml:5720]: updated with the latest commit, added the `/v21.0/me`

File: docs-v2/integrations/all/whatsapp-business.mdx (hunk 1 above)
Comment (hassan254-prog) [reply docs-v2/integrations/all/whatsapp-business.mdx]: The standard WhatsApp doesn’t officially offer a REST API for general or business use. Only the WhatsApp Business API, designed specifically for business interactions, provides a REST interface.

Comment (bodinsamuel) [reply docs-v2/integrations/all/whatsapp-business.mdx]: sorry I meant the A in App

Comment (hassan254-prog) [reply docs-v2/integrations/all/whatsapp-business.mdx]: I am sorry for missing that 🤦‍♂️, all good now fixed with [1bcc238](https://github.com/NangoHQ/nango/pull/2903/commits/1bcc238d57449631627b1aa162415cff24438b53)

**Review by bodinsamuel — APPROVED** [review]

**Pull request merged**